index.js

```diff
@@ -320,7 +320,7 @@ let Fixtures = {
             let request = {
                 url: '/' + VERSION + '/' + model.module,
                 method: 'POST',
-                data: model.attributes || {},
+                data: _.cloneDeep(model.attributes) || {},
```

Comment:
In order to make it clearer for everyone, why we need the code above, here's an example that illustrates that:

Remove said code.

Copy-paste the code below [here](https://github.com/sugarcrm/thorn/pull/134/files#diff-4663341696a08eef2606d83e7fb282a9R198)
```
console.log('%j', cmpFixture.attributes);
console.log('%j', falsyFixture.attributes);
console.log('%j', request.data);
```

Run the tests and you'll get something like:
```
{"name":"TestRecord1","testField1":"TestField1data","testField2":"TestField2data","testField3":false,"testField4":0}
{"name":"TestRecord1","testField1":"TestField1data","testField2":"TestField2data","testField3":false,"testField4":0,"testField5":"01z64fu6di8tzldnr5r05wj0p7wctv"}
{"name":"TestRecord1","testField1":"TestField1data","testField2":"TestField2data","testField3":false,"testField4":0,"testField5":"01z64fu6di8tzldnr5r05wj0p7wctv"}
```

Note that the two objects at the bottom are exactly the same.